src/types.ts

```diff
@@ -461,11 +461,12 @@ export interface FeeWindowRow {
 }
 
 export interface InitialReportersRow {
-  initialReportId: number;
   marketId: Address;
   reporter: Address;
-  isDesignatedReporter: boolean;
   amountStaked: number;
+  initialReporter: number;
+  redeemed: boolean;
+  isDesignatedReporter: boolean;
```

Comment:
Doesn't `repBalance` also need to be added to this interface?

Reply:
Yup!